rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -323,6 +323,67 @@ public void handmade_release_after_rskip_146_rejected_contractCaller() throws IO
         }
     }
 
+    @Test
+    public void release_before_rskip_219() throws IOException {
```

Comment:
You are including the creation of the BTC Release tx (when you call updateCollections) which is not needed to cover this test case.
Please remove the additional steps to keep the test sound and clear.